src/editions/transitioning.md

```diff
@@ -1,11 +1,100 @@
 # Transitioning your code to a new edition
 
-In general, to update your code to a new edition, the process
-will look like this:
+Transitioning between editions is built around lints. Fundamentally, the
+process works like this:
 
-* Make sure your build is warning-free.
-* Update or add the `edition` key in `Cargo.toml`
-* Fix any warnings, possibly by running `rustfix`.
+* Get your code compiling with no warnings.
+* Opt in to the new edition.
+* Fix any new warnings that may result.
 
-At this time, the `edition` key does not yet work, and `rustfix` is in a
-larval state. This section will update as our story gets better here.
\ No newline at end of file
+Luckily, we've been working on a tool to help assist with this process,
+`rustfix`. It can take suggestions from the compiler and automatically
+re-write your code to comply with new features and idioms.
+
+> `rustfix` is still quite young, and very much a work in development. But it works
+> for the basics! We're working hard on making it better and more robust, but
+> please bear with us for now.
+
+## Installing rustfix
+
+You can get `rustfix` from crates.io. Given that you're probably using Cargo,
+you'll want to run this:
+
+```shell
+$ cargo install cargo-fix --git https://github.com/rust-lang-nursery/rustfix
+```
+
+And that's it!
+
+## Prepare for the next edition
+
+Before we talk about how to move to the new edition, a reminder:
+
+
+* New editions may have features that are only available in the new edition.
```

Comment:
> New editions may have features that are only available in the new edition.

That sounds… obvious. How about

> New editions may introduce features that are available in that form in previous editions.

Reply:
This is what @alexcrichton suggested above; the opposite is also true, idk 😆

Reply:
Lol, true. Okay, how about this: "New editions might change the way you write Rust -- the add syntax, language, and library features but also remove others."